packages/@nimbus-js/runtime/tsconfig.json

```
@@ -1,4 +1,10 @@
 {
     "extends": "../../../tsconfig.json",
-    "include": ["src/**/*"]
+    "include": ["src/**/*"],
+
+    "compilerOptions": {
+        "noEmit": true,
+        "allowJs": true,
+        "typeRoots": ["./src/", "./node_modules/@types", "./node_modules/**"]
```

Do you need the `node_modules/**` here?

```suggestion
        "typeRoots": ["./src/", "./node_modules/@types"]
```